src/components/implementation/no_interface/scheddev/sl_lock.c

```diff
@@ -0,0 +1,67 @@
+#include "sl.h"
+#include "sl_lock.h"
+
+void
+sl_lock_init(struct sl_lock *lock)
+{
+    assert(lock);
+    *lock = SL_LOCK_STATIC_INIT();
+}
+
+thdid_t
+sl_lock_holder(struct sl_lock *lock)
+{
+    assert(lock);
+    return lock->holder;
+}
+
+
+void
+sl_lock_lock(struct sl_lock *lock)
```

Comment:
I recommended changing the name.  You didn't respond, and just did another PR without acknowledging the suggestion.

So now I'm making it mandatory:  Change the name as described previously.